src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,105 @@
 package core.basesyntax;
 
+import java.util.Arrays;
```

Comment:
Don't use Objects, Arrays, or any other util class. Instead of using `Arrays.copyOf`, you should implement a method to manually copy array elements.